backend/account/migrations/0002_auto_20230718_1040.py

```diff
@@ -1,30 +1,31 @@
 # mypy: ignore-errors
```

Comment:
There is no functionality changes here, These are only cleanup. The variable names are changed due to sonar issues.

Reply:
@muhammad-ali-e OK. For a fresh setup, this would work theoretically.

However what about an existing setup like Staging or Prod? I don't think Django will pick up changes in already done migrations. That is not recommended either.

Reply:
@hari-kuriakose Django will not re-run an migration that already processed. Here My inetntion is to remove those hardcoded values from the migration file  and for the fresh setup they can use the username and password from the ENV.

If we create a new migration file without altering the file those hardocded values still be there in repo.

Reply:
@hari-kuriakose reverted those changes.

Reply:
@muhammad-ali-e Sorry for the confusion. Let me rephrase this, for this specific scenario.

- Regarding **fresh installations** - Keep the changes to modify existing migration file, this will remove the hard coded values as you mentioned. Agree here.
- Regarding **updating existing installations** - Are we going to add a new data migration or instead mention in the release notes to do a one-time manual db update to change password?

Reply:
Here, we've opted to implement the changes outlined in the migration files through a separate PR, enhancing our ability to track and maintain a clear history.
cc: @hari-kuriakose

Reply:
Tagging the PR here:  https://github.com/Zipstack/unstract/pull/214